assets/src/edit-story/elements/text/edit.js

```diff
@@ -85,7 +92,9 @@ function TextEdit({
   },
   box: { x, y, height, rotationAngle },
 }) {
-  const { font } = rest;
+  const { font, fontSize } = rest;
+  const isItalic = useMemo(() => getHTMLInfo(content).isItalic, [content]);
```

Comment:
The same: let's combine memos. If you'd like, you can just create a local hook in the `text/` package, e.g. `useEnsureFontForElement(element)`.